packages/pluggableWidgets/radio-buttons-native/src/RadioButtons.editorConfig.ts

```diff
@@ -6,14 +6,15 @@ import lightUnselectedRadioIcon from "./assets/radioButton_unselected_light.svg"
 import darkUnselectedRadioIcon from "./assets/radioButton_unselected_dark.svg";
 
 export function getPreview(values: RadioButtonsPreviewProps, isDark: boolean): StructurePreviewProps {
+    const fontColor: string = isDark ? "#D6D6D6" : "#0A1324";
     const radioButtonLabel: StructurePreviewProps = {
         type: "Container",
         borders: false,
         children: [
             {
                 type: "Text",
-                content: values.label !== "" ? values.label : "Label",
-                fontColor: isDark ? "#D6D6D6" : "#0A1324"
+                content: values.label,
```

Comment:
Howcome you changed this expression? It handled empty string differently, right?

Reply:
Okay